config/operations.go

```diff
@@ -0,0 +1,56 @@
+package config
+
+import (
+	"github.com/layer5io/meshery-adapter-library/adapter"
+)
+
+const (
+
+	// Install Operation Commands
+	InstallBookInfoCommand = "install_book_info"
+	InstallHTTPBinCommand  = "install_http_bin"
+	InstallImageHubCommand = "install_image_hub"
+
+	// Custom Operation Commands
+	CustomOpCommand = "custom"
+)
+
+var (
+	SampleOperations = adapter.Operations{
```

Comment:
@mgfeller  Im changing `SampleOperations` to `CommonOperations`. Im intending to have these here because there are a set of operations that is common across adapters which is generic enough. And every adapter can opt for having those included or not by using thee `CommonOperations` construct. This is not an example but more of something out of the box for using directly, does this make sense now? @mgfeller